src/index.js

```diff
@@ -0,0 +1,75 @@
+import {
+  searchbar,
+  searchbutton,
+  name,
+  temp,
+  desc,
+  humidity,
+  body,
+  tempswitchBtn,
+} from './dom';
+
+import getweather from './getdata';
+
+const emptyresutl = () => {
+  desc.innerHTML = '';
+  name.innerHTML = '';
+  temp.innerHTML = '';
+  humidity.innerHTML = '';
+  tempswitchBtn.className = 'none';
+};
+
+const tempswitch = (val, type) => {
+  let temp = null;
+  if (type === '℃') {
+    temp = (val * 9) / 5 + 32;
+    type = '℉';
+  } else {
+    temp = (val - 32) * (5 / 9);
+    type = '℃';
+  }
+  return { temp, type };
```

Comment:
- [ ] [OPTIONAL] Floats are problematic to calculate with, and also take a lot of space, try to show the users only two decimal houses, ex: `19.22` instead of `19.222222222`.
[Cool article on using toFloat for that.](https://blog.udemy.com/javascript-parsefloat/?utm_source=adwords&utm_medium=udemyads&utm_campaign=DSA_Catchall_la.EN_cc.ROW&utm_content=deal4584&utm_term=_._ag_88010211481_._ad_437497337004_._kw__._de_c_._dm__._pl__._ti_dsa-406594358574_._li_1031670_._pd__._&matchtype=b&gclid=Cj0KCQjwwOz6BRCgARIsAKEG4FUATmnainxlYGoeo6HbAW5Rfzo-nNgNe6ruGvOmNKlBShj5IoCTxMkaApmEEALw_wcB)
![Screenshot_2020-09-11 Weather](https://user-images.githubusercontent.com/16193212/92931183-74d99e00-f419-11ea-819e-c2464ba8d0cc.png)